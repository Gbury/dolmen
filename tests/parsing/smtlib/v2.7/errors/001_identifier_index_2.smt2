(set-logic ALL)
(assert (as (as) bar))
