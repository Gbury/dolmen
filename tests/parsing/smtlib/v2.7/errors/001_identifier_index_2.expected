File "tests/parsing/smtlib/v2.7/errors/001_identifier_index_2.smt2", line 2, character 13-15:
2 | (assert (as (as) bar))
                 ^^
Error while parsing an identifier, read the reserved word 'as',
      but expected an underscore: identifiers starting with an opening
      parenthesis must be indexed identifiers, of the form "(_ symbol
      index+)".
